travis/scripts/05-run.sh

```diff
@@ -124,3 +124,35 @@ if [ "$RUN_APP" == 1 ]; then
     kill $(cat .pid)
     exit $result
 fi
+
+#-------------------------------------------------------------------------------
+# Run Docker
+#-------------------------------------------------------------------------------
+if [ "$RUN_APP" == 1 ]; then
+    if [[ "$PROFILE" == "prod" ]]; then
+        cd "$APP_FOLDER"
+        if [ -f "mvnw" ]; then
+            ./mvnw -DskipTests package jib:dockerBuild -P"$PROFILE"
+        elif [ -f "gradlew" ]; then
+            ./gradlew bootWar jibDockerBuild -P"$PROFILE" -x test
+        else
+            echo "No mvnw or gradlew"
+            exit 0
+        fi
+        if [ $? -ne 0 ]; then
+            echo "Error when building Docker image"
+            exit 1
+        fi
+
+        cd "$APP_FOLDER"
+        docker-compose -f src/main/docker/app.yml up -d
+        sleep 40
```

Comment:
I'm afraid this will slow down our Travis build...

Reply:
I think for this kind of tests, it should go to daily builds at https://github.com/hipster-labs/jhipster-travis-build
See https://travis-ci.org/hipster-labs/jhipster-travis-build/branches

Reply:
@pascalgrimaud :+1: but I don't understand the branch organization (the docker branch is not synchronized with the master).
I'll remove the above `docker run` snippet to move to jhipster-travis-build